ui/components/ui/box/box.js

```diff
@@ -313,11 +313,15 @@ Box.propTypes = {
     PropTypes.arrayOf(PropTypes.oneOf(Object.values(BLOCK_SIZES))),
   ]),
   backgroundColor: MultipleBackgroundColors,
-  color: MultipleTextColors,
   className: PropTypes.string,
   /**
    * The polymorphic `as` prop allows you to change the root HTML element of the Box component
    * Defaults to 'div'
    */
   as: PropTypes.string,
+  /**
+   * The color of the Typography component Should use the COLOR object from
+   * ./ui/helpers/constants/design-system.js
+   */
+  color: ValidTextColors,
```

Comment:
Update this to `MultipleTextColors`

Reply:
Resolved in https://github.com/MetaMask/metamask-extension/pull/15565/commits/d5e2c376ee744930742729758618aacf20167d39